### PR TITLE
Update CHANGE_LOG for 0.52.0

### PR DESCRIPTION
**Pull request by stuartarchibald**
As title.

### CODE REVIEW

**Review by gmarkall — APPROVED** [review]
This looks good to me!

**Review by sklam — COMMENTED** [review]
Just a minor thing

**Review by sklam — APPROVED** [review]

**Comment by sklam** [comment]
Just realize it is missing 6275, 6319

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,7 +1,241 @@
...
+* PR #6304: Support NumPy 1.19 (Stuart Archibald)
+* PR #6309: Add suitable file search path for BSDs. (Stuart Archibald)
+* PR #6341: Re roll 6279 (Rishabh Varshney and Valentin Haenel)
+
+Performance Enhancements:
+
+* PR #6145: Patch to fingerprint namedtuples. (Stuart Archibald)
+* PR #6202: Speed up str(int) (Stuart Archibald)
+* PR #6261: Add np.ndarray.ptp() support. (Stuart Archibald)
+* PR #6266: Use custom LLVM refcount pruning pass (Siu Kwan Lam)
```
Comment (sklam) [inline CHANGE_LOG:90]:
```suggestion
* PR #6266: Use custom LLVM refcount pruning pass (Siu Kwan Lam)
* PR #6275: Switch on SLP vectorize. (Stuart Archibald)
```

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,7 +1,241 @@
...
+* PR #6265: Fix #6260 (Stuart Archibald)
+* PR #6267: speed up a couple of really slow unittests (Stuart Archibald)
+* PR #6281: Remove numba.jitclass shim as per deprecation schedule. (Stuart
+  Archibald)
+* PR #6294: Make return type propagate to all return variables (Andreas Sodeur)
+* PR #6300: Un-skip tests that were skipped because of #4026. (Owen Anderson)
+* PR #6307: Remove restrictions on SVML version due to bug in LLVM SVML CC
+  (Stuart Archibald)
+* PR #6316: Make IR inliner tests not self mutating. (Stuart Archibald)
+* PR #6318: PR #5892 continued (Todd A. Anderson, via Stuart Archibald)
```
Comment (sklam) [inline CHANGE_LOG:132]:
```suggestion
* PR #6318: PR #5892 continued (Todd A. Anderson, via Stuart Archibald)
* PR #6319: Permit switching off boundschecking when debug is on. (Stuart Archibald)
```

**Pull request merged**